samples/dotnet/KernelHttpServer/Program.cs

```diff
@@ -22,9 +28,12 @@ public static void Main()
 
                 var builtConfig = config.Build();
             })
-            .ConfigureServices(services =>
+            .ConfigureServices(async services =>
             {
-                services.AddSingleton<IMemoryStore<float>>(new VolatileMemoryStore());
+                //services.AddSingleton<IMemoryStore<float>>(new VolatileMemoryStore());
+
+                await dbConnection.OpenAsync();
```

Comment:
why not open the connection on demand, and close it, rather than keeping it open forever?